example/lib/main.dart

```diff
@@ -7,7 +7,7 @@ void main() {
     activateLicense();
 
     final status = LexActivator.IsLicenseGenuine();
-    LexActivator.SetCallback(callback: (status) {
+    LexActivator.SetLicenseCallback(callback: (status) {
       print('Callback status: $status');
```

Comment:
@mumin-khan Maybe we can improve this to print different messages for different cases. Check other license callback functions of other wrappers.